syntaxes/GDScript.tmLanguage.json

```diff
@@ -274,6 +274,11 @@
 					"match": "[-]?([0-9][0-9_]*e[\\-\\+]?\\[0-9_])",
 					"name": "constant.numeric.float.gdscript"
 				},
+				{
+					"name": "constant.numeric.float.gdscript",
+					"match": "(?x)\n  (?<! \\w)(?:\n    (?:\n      \\.[0-9](?: _?[0-9] )*\n      |\n      [0-9](?: _?[0-9] )* \\. [0-9](?: _?[0-9] )*\n      |\n      [0-9](?: _?[0-9] )* \\.\n    ) (?: [eE][+-]?[0-9](?: _?[0-9] )* )?\n    |\n    [0-9](?: _?[0-9] )* (?: [eE][+-]?[0-9](?: _?[0-9] )* )\n  )([jJ])?\\b\n",
+					"captures": { "1": { "name": "storage.type.imaginary.number.gdscript" } }
```

Comment:
Imaginary number with `j` suffix? Are complex numbers supported syntax in GDScript now?
Is there a source proposal/docs or something?

Reply:
Nope, I copied it from MagicPython and didn't notice the j's! Thanks for the catch.

Reply:
You probably want to double-check by testing whether it supports, e.g. lower and uppercase `e`, `E` and `f`, `F` suffixes. Also whether it supports `_` separators and `.` cases like `0.` or `.5`. Float literals syntax can vary a lot depending on the language, e.g. GDShader (or whatever it's going to be called) technically accepts even weird cases like `0.f` IIRC.